tests/unit/lib/subscription-type.test.js

```diff
@@ -0,0 +1,24 @@
+const { loadSchema } = require("@graphql-tools/load");
+const { GraphQLFileLoader } = require("@graphql-tools/graphql-file-loader");
+const { getFilteredTypeMap } = require("../../../src/lib/graphql");
+
+const SCHEMA_FILE = require.resolve("../../__data__/subscription.graphql");
+
+describe("subscription object", () => {
+  let schema;
+  beforeAll(async () => {
+    schema = await loadSchema(SCHEMA_FILE, {
+      loaders: [new GraphQLFileLoader()],
+    });
+  });
+
+  test("returns a filtered map of schema types", () => {
+    const schemaTypeMap = getFilteredTypeMap(schema.getTypeMap());
+
+    expect(schemaTypeMap).toEqual(
+      expect.objectContaining({
+        Subscription: expect.any(Object),
+      }),
+    );
```

Comment:
I recommend using the plain object match with `toMatchInlineSnapshot` as it gives a complete overview of the GraphQL schema - thanks to the latest Jest, those snapshots are now easier to deal with.

```suggestion
    expect(schemaTypeMap).toMatchInlineSnapshot(`
      {
        "Boolean": "Boolean",
        "ID": "ID",
        "String": "String",
        "Subscription": "Subscription",
      }
    `);
```